_posts/en/2021-12-10-iitb-diagnostic-study.md

```diff
@@ -0,0 +1,139 @@
+
+## Introspection
+
+Underlying and persistent organizational issues tend to lead to emotional exhaustion throughout an organization; however, recognizing the issue and seeking help can make a difference.
+Organizational issues can be a challenge and the first step to resolve them is to acknowledge where a problem is and identify the source.
+It may take time to determine where issues are coming from, but identifying and resolving these issues is highly beneficial to both the organization and its employees.
+The quicker organizational issues are resolved, the sooner a healthier workspace can be created.
+
+The harshest restrictions of the pandemic may be behind us, but many employees feel more and more isolated and mentally drained.
+Better communication helps give staff the support they need to improve their wellbeing.
+Clear communication is central to making the next unstable phase a success.
+With the **goal of boosting communication identifying major blockers and areas of improvement in our branch**, we sought to go interview the directors of the IITB and hear their stories. We talked with directors across IITB in a series of one-on-one interviews to better understand their day-to-day experience within the branch and their views of ESDC as a whole.  
+
+## Why conduct interviews rather than send out a survey
+
+Before we talk about how we prepared for the interviews, let me be honest with you: interviews aren’t the easiest route to take.
+Especially in our case, knowing how overwhelming the work environment of directors has always been.
+First, we had the idea of sending a survey to the directors and doing our analysis based on the data we collected from the survey, but the intent and goal of our study was different than what a survey would have provided.
+
+We wanted to make interviewees feel more comfortable expressing their true feelings and opinions and to give us as interviewers the opportunity to dive into specific topics.
+Talking to people will always tell more than surveying, and survey fatigue was one of our concerns given the number of surveys and communications we are presented with in the branch.
+
+## What was the goal
+
+We wanted **to uncover directors' and their teams’ pain points, highlight success stories and innovative ideas**.
+By doing so, we wanted to collect information that would inform how we can invest in our internal capabilities to improve IITB service delivery, and to develop upcoming plans and strategies.
+You might ask, ‘"why would you do this?”’ as other reports have been completed: [Price Waterhouse Coopers (PwC) study](https://www.pwc.com/ca/en/today-s-issues/upskilling/canadian-pulse-survey.html) conducted in 2020, [the Gartner iScore report](https://www.gartner.com/en).
```

Comment:
Please don't share...we are not legally allowed to.  We can speak to them, but not share.

Reply:
Two other reports were written but third party firms at ESDC to help assess its IM/IT functions.

Deux autres rapports ont été rédigés par des entreprises tierces à EDSC pour aider à évaluer ses fonctions de GI/TI.